src/theory/quantifiers/sygus/sygus_grammar_cons.cpp

```diff
@@ -794,10 +794,10 @@ void CegGrammarConstructor::mkSygusDefaultGrammar(
         // make data type for positive constant integers
         sdts.push_back(SygusDatatypeGenerator(pos_int_name));
         /* Add operator 1 */
-        Trace("sygus-grammar-def") << "\t...add for 1 to Pos_Int\n";
+        Trace("sygus-grammar-def") << "\t...add for 1.0 to Pos_Int\n";
```

Comment:
This whole block should be adapted to not refer to ints but to rationals, I think. The names and docs are all wrong.